code/socialDistribution/urls.py

```diff
@@ -9,9 +9,9 @@
   path('logout/', views.logoutUser, name='logout'),
   path('authors/', views.authors, name='authors'),
   path('author/<int:author_id>/', views.author, name='author'),
-  path('author/<int:author_id>/home/', views.home, name='home'),
+  path('home/', views.home, name='home'),
```

Comment:
can you please update line 12 in code/socialDistribution/decorators.py from `return redirect('socialDistribution:home', author_id=author_id)` to `return redirect('socialDistribution:home')` since our home route doesn't expect the author id anymore